packages/iframe-coordinator-cli/package.json

```diff
@@ -10,7 +10,6 @@
     "express": "^4.21.1",
     "find-root": "^1.1.0",
     "http-proxy-middleware": "^3.0.3",
-    "iframe-coordinator": "6.2.0",
```

Comment:
This dependency is not needed because NPM/vite will automatically look up imports of `iframe-coordinator` from the other packages in the workspace. The Vite build then bundles all of the IFC code into a single js file in `dist/`, such that it's also not a required dependency when installing ifc-cli in other projects.